composer.json

```diff
@@ -4,19 +4,23 @@
     "keywords": ["doctrine", "aes256", "encrypt", "decrypt"],
     "license": "MIT",
     "description": "Package encrypts and decrypts Doctrine fields through life cycle events.",
-
     "require": {
-        "php": ">=5.3.2",
-        "doctrine/orm": "~2.0",
-        "ext-mcrypt": "*"
+        "php": "^7.1",
+        "doctrine/orm": "^2.5"
     },
-
     "require-dev": {
-        "phpunit/phpunit": "~3.7"
+        "illuminate/encryption": "^5.4",
```

Comment:
Can you use a lower version here as a minimum? Also, given that there is only 1 encryptor implementation this should probably go into the require section. As a minimum you should add a suggest section containing this so people know they need to install it to have this work.